### DISCUSSION TITLE
op.h file missing

### DISCUSSION THREAD

**Discussion by Keezuu** [apache/incubator-mxnet#19593] [Q&A] [answered]
Hi, I'm trying to build the MXNet code for android platform, unfortunately when I do this using the Dockerfile.build.android to create the environment and build_android_armv7 function from runtime_functions.sh inside the environment I do not get the required **op.h** header file generated. Same happens if I want to just build it for ubuntu cpu, the USE_CPP_PACKAGE flag is set as "ON" but no **op.h.** header is generated even though the build is successful. Is there something specific that needs to be done to generate that header?

**Comment by szha** [comment, marked as answer]
Thanks for reporting the problem. Are you using the v1.x branch? USE_CPP_PACKAGE currently takes no effect in master branch as the C++ API will be refactored for integrating with the new FFI.

**Reply by Keezuu** [reply]
Thanks for answer. Using v1.x branch docker files and files for build did actually help :) So to use the C++ API right now one has to use the v1.x branch because it's impossible to generate "op.h" for c++ code from master branch?

**Reply by szha** [reply]
that's correct. once c++ interface for master branch completes, the interfaces in op.h will be reintroduced.
